azure-pipelines-build-devtest.yml

```diff
@@ -40,8 +40,8 @@ resources:
 
 extends:
   # Filename in Azure DevOps Repository (note possible -ui or -api)
-  # Django example: azure-pipelines-linkedevents-api-release.yml
+  # Django example: azure-pipelines-PROJECTNAME-api-release.yml
   # Drupal example: azure-pipelines-drupal-release.yml
-  template: azure-pipelines-build-linkedcomponents-ui.yml@linkedevents-pipelines
+  template: azue-pipelines-build-linkedcomponents-ui-devtest.yml@linkedevents-pipelines
```

Comment:
Is the naming intentional or should it be: azure-pipelines-build-linkedcomponents-ui-devtest.yml@linkedevents-pipelines (Missing r in azure)